VideoLocker/res/layout/view_progress_download_btn.xml

```diff
@@ -15,16 +14,9 @@
         android:layout_width="25dp"
         android:layout_height="25dp"
```

Comment:
these are standardized and should be `24dp`. #434 introduced standard sizes from the design doc. These should translate to `@dimen/fa_xx_large`.
p.s. you'll have to rebase your branch for the latest dimen resources.